Plugins/Disk/Retrieve-FLSBody.ps1

```diff
@@ -91,12 +91,17 @@ process{
     
     Invoke-Command -Session $Session -ScriptBlock $ScriptBlock
 
+    #Compress output before copying to analysis machine

```

Comment:
Do we want to split the `fls.exe ...` and `Compress-Archive` commands into multiple `Invoke-Command` calls? Or use one script block for both?